tests/ui/lint/lint-qualification.rs

```diff
@@ -1,6 +1,6 @@
 //@ run-rustfix
 #![deny(unused_qualifications)]
-#![allow(deprecated)]
+#![allow(deprecated, dead_code)]
```

Comment:
This should be unnecessary because all `unused` lints, including `dead_code`, are allowed by default in ui tests.

Reply:
It isn't passed when compiling the `.fixed` files

```
--- stderr -------------------------------
warning: function `f` is never used
  -->  ..../lint-qualification.fixed:41:8
   |
LL |     fn f() {
   |        ^
   |
   = note: `#[warn(dead_code)]` on by default

warning: 1 warning emitted
------------------------------------------
```